engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -508,4 +521,20 @@ public State emptyState() {
   public int getMaxUnboxingLayouts() {
     return 10;
   }
+
+  private <T> T getOption(OptionKey<T> key) {
+    var options = getEnvironment().getOptions();
+    var safely = false;
+    assert safely = true;
```

Comment:
Many `runtime-with-instruments` tests fail.

Reply:
Shouldn't we rather create another pivotal for that, at least? Rather than having yet another weird hack. The option descriptor registration, and processing of options should be addressed sooner or later anyway.

Reply:
Reported as [pivotal](https://www.pivotaltracker.com/story/show/184333049).